extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -497,7 +497,7 @@ export class AmpAnalytics extends AMP.BaseElement {
    */
   generateRequests_() {
     if (!this.config_['requests']) {
-      if (!this.isInabox_) {
+      if (!this.allowParentPostMessage_) {
```

Comment:
`allowedParentPostMessage_` is a method. Please use `this.allowParentPostMessage_()` Thank you.

Reply:
Done, sorry about that.